code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1879,4 +1879,43 @@
 		if(added_length >= 0.20) //Only add the length if it's greater than or equal to 0.2. This is to prevent people from smoking the reagents and causing the penis to update constantly.
 			P.length += added_length
 			P.update()
-	..()
\ No newline at end of file
+	..()
+
+/datum/reagent/changeling_string
```

Comment:
Dont forget to black list this from plants if you havnt already

Reply:
Where is the file that handles plant blacklisting?

Reply:
If it's temporary, wouldn't it be fine to appear in plants?

Reply:
The issue is that on its own, chemical does literally nothing so it would be a bad idea to include it because it's a waste.